kong/db/dao/plugins/go.lua

```diff
@@ -444,18 +471,22 @@ end
 local get_plugin do
   local loaded_plugins = {}
 
+  local function get_plugin_info(name)
+    local cmd = string.format(
+        "bin/go-pluginserver -plugins-directory %q -dump-plugin-info %q",
```

Comment:
looks like this also needs the locatiion of `go-pluginserver`

Reply:
duh!  done